src/components/header/NavDropDownButton/NavDropDownButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+type NavDropDownButtonProps = {
+  label: string
+  id: string
```

Comment:
🎨 This is a place where I would add a comment documenting the prop. This `id`should not the id of this button (as a someone might expect glancing at prop types) but be the id of some other element  already present in the DOM will be controlled by this button.

Reply:
I'd even suggest renaming this prop (perhaps `menuId`?) so that the user _can_ pass in an `id` attribute to this element if they need to